docs/quick-start/configure-contracts.mdx

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_position: 4
+description: How to Configure Contracts.
+---
+
+# Configure Contracts
```

Comment:
Please change this into "Importing other contracts" (i know it's named configure contracts but this is the better title for the docs i feel), since the function of this component is to import existing contracts into the application.